templates/vim-for-vscode.txt

```diff
@@ -0,0 +1,26 @@
+Learning Goal: Learn how to use Vim with VS Code
+
+Learning Curriculum: https://vimforvscode.com/
+
+Special Requirements:
+- VSCode
```

Comment:
They also need a license to the course right?

Reply:
oh good call! yes, I will add that in